app/helpers/compute_resources_vms_helper.rb

```diff
@@ -212,7 +212,7 @@ def vm_delete_action(vm, authorizer = nil)
 
   def vsphere_scsi_controllers(compute_resource)
     scsi_controllers = {}
-    compute_resource.scsi_controller_types.each { |type| scsi_controllers[type[:key]] = type[:title] }
+    compute_resource.storage_controller_types.each { |type| scsi_controllers[type[:key]] = type[:title] }
     scsi_controllers
```

Comment:
Again, already a problem before but I think this whole method can be implemented as:
```ruby
compute_resource.storage_controller_types.to_h { |type| [type[:key], type[:title]] }
```

Reply:
I don't see this method being called anywhere. Not sure what it's for.

Reply:
Excellent observation. It was added in 80dc430ed4b45794ff25c514ad03a5d3d45d2135 but in that commit I don't see it used and don't see it elsewhere either. Removal makes sense to me.